cloud/terraform/terraform_configurator.py

```diff
@@ -70,10 +75,26 @@ def __new_aws_instance(self, instance):
             'instance_type': instance['instance_type'],
             'ami': instance['ami'],
             'provider': f'aws.{instance["region"]}',
+            'key_name': f'{instance["region"]}-key',
             'tags': {'name': name},
         }
         self.resources_tf['resource']['aws_instance'][name] = new_instance
 
+    def add_ssh_key_conf(self, region):
+        if self.cloud == 'aws':
+            self.__new_aws_key_pair(region)
+
+    def __new_aws_key_pair(self, region):
+        key_name = f'{region}-key'
+        
+        new_key_pair = {
+            'provider': f'aws.{region}',
+            'key_name': key_name,
+            'public_key': f'${{file(\"{self.ssh_key_path}.pub")}}',
```

Comment:
Q: Do we need to escape the first double quotes?

Reply:
Yes, if not, everything will get messed up when we dump this to the json

Reply:
So, should we escape the second double quotes?
```suggestion
            'public_key': f'${{file(\"{self.ssh_key_path}.pub\")}}',
```

Reply:
Okay, I tried again and we don't need to scape the double quotes 🤷